contrib/internal/httputil/trace_gen.go

```diff
@@ -0,0 +1,119 @@
+// Code generated by ./gen do not modify
```

Comment:
Let's respect the standard comment heading for generated files outlined here: https://github.com/golang/go/issues/13560#issuecomment-276866852